store/sqlstore/user_store.go

```diff
@@ -1039,18 +1051,18 @@ func (us SqlUserStore) PermanentDelete(userId string) store.StoreChannel {
 
 func (us SqlUserStore) Count(options model.UserCountOptions) store.StoreChannel {
 	return store.Do(func(result *store.StoreResult) {
-		query := sq.Select("COUNT(Users.Id)").From("Users")
+		query := sq.Select("COUNT(DISTINCT u.Id)").From("Users AS u")
```

Comment:
Hey @jespino - I ran into this code while doing something else. Curious to know if we need the "DISTINCT" function here? Id is already a primary key of the users table. And I don't see joins with the same table, or any unions of sorts.

Reply:
It is hard to try, but as part of the "ApplyViewRestrictions" function to the query, it joins with the team members and the channel members, so I guess the `DISTINCT` was added there to avoid duplicated rows based on that.

Reply:
Hmm .. okay but you are joining with the user id. So there shouldn't be any duplicated rows right?